stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -200,11 +200,35 @@ private func configureRuntimeFromEnvironment() {
       return
     }
 
-    guard address.prefix(7) == "grpc://" else {
-      fatalError("SWIFT_TENSORFLOW_SERVER_ADDRESS must start with 'grpc://'.")
+    if let idx = address.firstIndex(of: ":") {
+      if let endIdx = address.index(idx, offsetBy: 3, limitedBy:address.endIndex) {
+        if address[idx..<endIdx] == "://" {
+          let `protocol` = address[address.startIndex..<idx]
```

Comment:
what values can `protocol` take? Please add as a comment.

also, if there is a small set of legal values, consider CHECK on them.

Reply:
Publicly only grpc, but I want to be able to add other values for development.